refresh_test.py

```diff
@@ -76,12 +76,15 @@ def nodetool_refresh(self, node):
             time.sleep(int(self.params.get('flush_period', default=60)))
             node.run_nodetool("flush")
 
-    def check_timeout(self):
+    def check_timeout(self):  # pylint: disable=too-many-locals
         assert self.monitors.nodes, 'Monitor node should be set, we will try to get metrics from Prometheus server'
-        cmd = 'curl http://[%s]:9090/api/v1/query_range?query=scylla_storage_proxy_coordinator_read_timeouts&start=%s&end=%s&step=60s' % (
-            self.monitors.nodes[0].external_address, self.start, self.end)
+        base_url = "http://%s:9090/api/v1/query_range" % normalize_ipv6_url(self.monitors.nodes[0].external_address)
+        range_str = "&start={0.start}&end={0.end}".format(self)
+        cmd = ["curl",
```

Comment:
why this test used curl ? and request ? seem like a very old test that wasn't run in a while...

Reply:
yes, I think we need to remove it. There is a refresh nemesis instead